microsoft_intune/help.md

```diff
@@ -56,6 +56,39 @@ Example input:
 
 ### Actions
 
+#### Add application to a policy
+
+This action is used to add an application to a policy.
+
+##### Input
+
+|Name|Type|Default|Required|Description|Enum|Example|
+|----|----|-------|--------|-----------|----|-------|
+|application_name|string|None|True|Application name|None|Adobe Reader|
+|device_name|string|None|True|Device name|None|None|
+|device_type|string|None|True|Device type|['Android', 'iOS']|None|
+|policy_name|string|None|True|Policy name|None|my-android-app-policy|
+
+Example input:
+
+```
+{
```

Comment:
if all the inputs are `required: true` we should add the other two options in this JSON example